tensorizer/serialization.py

```diff
@@ -781,7 +785,7 @@ def __init__(
                 for name, entry in self._metadata.items()
             }
             self.total_tensor_bytes = sum(tensor_sizes.values())
-            self._plaid_mode_buffer_count = 4
+            self._plaid_mode_buffer_count = plaid_mode_buffers
```

Comment:
Continuation of a previous suggestion:
```suggestion
            self._plaid_mode_buffer_count = plaid_mode_buffers or 4
            if not self._plaid_mode and plaid_mode_buffers is not None:
                raise ValueError("Cannot specify plaid_mode_buffers when plaid_mode=False")
```